lib/llvmopencl/Workgroup.cc

```diff
@@ -1216,7 +1216,12 @@ Workgroup::createAllocaMemcpyForStruct(LLVMModuleRef M, LLVMBuilderRef Builder,
     args[1] = CARG1;
     args[2] = Size;
 
+#ifdef LLVM_OLDER_THAN_14_0
     LLVMValueRef call4 = LLVMBuildCall(Builder, MemCpy4, args, 3, "");
```

Comment:
I believe the LLVM style wants capital variable names. Can you fix this while you are touching the code around here?